potpie/context-engine/adapters/inbound/cli/templates/claude_plugin/commands/potpie-record.md

```
@@ -0,0 +1,35 @@
+Record durable Potpie learnings after useful work.
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)